inc/class-main.php

```diff
@@ -468,6 +469,23 @@ public function after_update_migration() {
 		return update_option( 'themeisle_blocks_db_version', OTTER_BLOCKS_VERSION );
 	}
 
+	/**
+	 * About page SDK
+	 *
+	 * @return array
+	 * @since  2.3.1
+	 * @access public
+	 */
+	public function about_page() {
+		return array(
+			'location'         => 'otter',
+			'logo'             => 'https://ps.w.org/otter-blocks/assets/icon-256x256.png?rev=2019293',
```

Comment:
@HardeepAsrani, is this ok to use as a logo source? I see that our logo is in the `.wordpress-org` folder -- Is it better to use that one?

Reply:
@Soare-Robert-Daniel We already have an Otter logo in assets folder, try `assets/images/logo-alit.png`